channeldb/graph.go

```diff
@@ -1752,6 +1752,11 @@ func (l *LightningNode) PubKey() (*btcec.PublicKey, error) {
 	return key, nil
 }
 
+// ColorHexCode returns the node's color in hex code format.
+func (l *LightningNode) ColorHexCode() string {
```

Comment:
Better to make this a utility method like `parseHexColor` I think

Reply:
sure, makes sense